localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
-d_cult_of_sunwell_adj;Sunwell;;;;;;;;;;;;;x
-d_old_ways;The Church of the True Ways;;Kirche der wahrer Wege;;;;;;;;;;;x
-d_old_ways_adj;Druidic;;druidisch;;;;;;;;;;;x
-d_wolf_cult_religion;The Wolf Cult;Le Culte du Loup;Kult des Wolfs;;;;;;;;;;;x
-d_wolf_cult_religion_adj;Wolf;;w�lfisch;;;;;;;;;;;x
-d_ursine;The Ursine Cult;Le Culte-Ours;Kult des B�ren;;;;;;;;;;;x
-d_ursine_adj;Ursine;;b�risch;;;;;;;;;;;x
-d_kaldorei_religion;The Priesthood of Elune;Le Clerg� d'�lune;Elunes Priesterschaft;;;;;;;;;;;x
-d_kaldorei_religion_adj;Elune's;;elunisch;;;;;;;;;;;x
-d_cult_of_loa;The Cult of Loa;Le Culte des Loas;Loa-Kult;;;;;;;;;;;x
+d_eastern_arathi;Go'shek;Go'shek;;;Go'shek;;;;;;;;;x
+d_eastern_arathi_adj;Go'shek;Go'shek;;;Go'shek;;;;;;;;;x
+d_cult_of_sunwell;The Sunwell Cult;Le Culte du Puits de soleil;Kult des Sonnenbrunnens;;Culto de la Fuente del Sol;;;;;;;;;x
+d_cult_of_sunwell_adj;Sunwell;Puits de soleil;sonnenbrunnens;;Fuente del Sol;;;;;;;;;x
+d_old_ways;The Church of the True Ways;Église des Vraies Voies;Kirche der wahrer Wege;;La iglesia de los Caminos Verdaderos;;;;;;;;;x
+d_old_ways_adj;Druidic;druidique;druidisch;;druídico;;;;;;;;;x
+d_wolf_cult_religion;The Wolf Cult;Le Culte du Loup;Kult des Wolfs;;El Culto del Lobo;;;;;;;;;x
+d_wolf_cult_religion_adj;Wolf;;wölfisch;;;;;;;;;;;x
+d_ursine;The Ursine Cult;Le Culte-Ours;Kult des Bären;;El Culto Ursino;;;;;;;;;x
+d_ursine_adj;Ursine;;bärisch;;;;;;;;;;;x
+d_kaldorei_religion;The Priesthood of Elune;Le Clergé d'Élune;Elunes Priesterschaft;;El sacerdocio de Elune;;;;;;;;;x
+d_kaldorei_religion_adj;Elune's;d'Élune;elunisch;;;;;;;;;;;x
+d_cult_of_loa;The Cult of Loa;Le Culte des Loas;Loa-Kult;;El Culto Loa;;;;;;;;;x
 d_cult_of_loa_adj;Loa;;loisch;;;;;;;;;;;x
-d_cult_of_hakkar;The Cult of Hakkar;Le Culte d'Hakkar;Hakkar-Kult;;;;;;;;;;;x
+d_cult_of_hakkar;The Cult of Hakkar;Le Culte d'Hakkar;Hakkar-Kult;;EL Culto Hakkar;;;;;;;;;x
 d_cult_of_hakkar_adj;Hakkari;;hakkarisch;;;;;;;;;;;x
-d_spider_philosophy;The Spider Council;Le Conseil-Araign�e;Rat der Spinne;;;;;;;;;;;x
-d_spider_philosophy_adj;Council;;r�tisch;;;;;;;;;;;x
-d_water_deities;The Church of the Tides;L'�glise des Flots;Kirche der Gezeiten;;;;;;;;;;;x
-d_water_deities_adj;Tide;;gezeitisch;;;;;;;;;;;x
-d_cult_of_the_candle;The Church of the Candle;Le Culte de la Cire;Kirche der Kerze;;;;;;;;;;;x
-d_cult_of_the_candle_adj;Wax;;w�chsern;;;;;;;;;;;x
-d_agamaggan_worship;The Cult of Agamaggan;Le Culte d'Agamaggan;Kult von Agamaggan;;;;;;;;;;;x
+d_spider_philosophy;The Spider Council;Le Conseil-Araignée;Rat der Spinne;;El Consejo de la Araña;;;;;;;;;x
+d_spider_philosophy_adj;Council;Conseil;rätisch;;Consejo;;;;;;;;;x
+d_water_deities;The Church of the Tides;L'Église des Marées;Die Kirche der Gezeiten;;Iglesia de las Mareas;;;;;;;;;x
+d_water_deities_adj;Tide;Marée;gezeitisch;;Marea;;;;;;;;;x
+d_cult_of_the_candle;The Church of the Candle;Le Culte de la Bougie;Die Kirche der Kerze;;La Iglesia de la Vela;;;;;;;;;x

```

Comment:
earlier (line 93) you deleted the article in front of "Schwarzes Imperium". Please be consistent, either add the article there again, or delete them here as well "Die Kirche ..." to "Kirche ..."

Reply:
There's no consistency currently. I follow the English translation.

In line 93 it says "Black Empire", in this one it says "The Church of the Candle".

This is something the GoA team needs to decide on, as I've changed nothing to the English translations and that's where I base the other ones from.